assets/prometheus-k8s/rules.yaml

```diff
@@ -334,6 +334,14 @@ spec:
     - expr: sum(rate(container_cpu_usage_seconds_total{container_name!="POD",container_name!="",pod_name!=""}[5m]))
         / sum(machine_cpu_cores)
       record: cluster:container_cpu_usage:ratio
+    - expr: sum(node:node_cpu_utilisation:avg1m)
+      record: cluster:node_cpu_utilisation:avg1m
+    - expr: sum(node:node_memory_utilisation:ratio)
+      record: cluster:node_memory_utilisation:ratio
+    - expr: sum(node_filesystem_size_bytes{fstype=~"ext[234]|btrfs|xfs|zfs"})
```

Comment:
This metric shows each filesystem even if the same device is mounted on multiple mountpoints. I think we're looking for:

```
sum(max by(instance, device) (node_filesystem_size_bytes{fstype=~"ext[234]|btrfs|xfs|zfs"}))
```